### PR TITLE
Choco package for v1.3.0

### PR DESCRIPTION
**Pull request by apoorvdeshmukh**
Updating files required to prepare a choco package.
Tested as below
Updated sqlcmd.nuspec and tools\chocolateyinstall and ran `choco pack sqlcmd.nuspec`
```
git\go-sqlcmd\release\windows\choco>choco pack sqlcmd.nuspec
Chocolatey v2.1.0
Attempting to build package from 'sqlcmd.nuspec'.
Successfully created package 'git\go-sqlcmd\release\windows\choco\sqlcmd.1.3.0.nupkg'
```

Tested as per instructions [here](https://docs.chocolatey.org/en-us/create/create-packages#testing-your-package)
No issues seen.

### CODE REVIEW

File: release/windows/choco/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 $url        = '{{DownloadUrl}}'
-$url64      = 'https://download.microsoft.com/download/d/4/4/d4403a51-2ab7-4ea8-b850-d2710c5e1323/sqlcmd_0.8.1-1.msi'
+$url64      = 'https://github.com/microsoft/go-sqlcmd/releases/download/v1.3.0/sqlcmd-x64_1.3.0-1.msi'
```
Comment (shueybubbles) [inline release/windows/choco/tools/chocolateyinstall.ps1:6]: changing from the downloads site to github seems like a big deal. Do you know why we were using downloads before, and if there's any approval needed to stop using it?

And what is the value of `$url` here?

**Comment by shueybubbles** [comment]
File an issue to change how these files are generated. We should just have a template in the repo, and the official build should create the package by populating the template with the actual tag value and github locations and checksum.

File: release/windows/choco/tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (apoorvdeshmukh) [reply release/windows/choco/tools/chocolateyinstall.ps1:5]: I checked the download site for the previous version [here](https://community.chocolatey.org/packages/sqlcmd#files) and noticed that it is pointing to github. Thought of updating the files in our repo first to keep them in sync with choco. The choco command to push the package doesn't really rely on these files.

**Comment by shueybubbles** [comment]
we shouldn't add nupkg files to the repo

File: release/windows/choco/sqlcmd.nuspec [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>sqlcmd</id>
-    <version>0.8.1</version>
+    <version>1.3.0</version>
```
[the remark this answers is not in the document]
Comment (shueybubbles) [reply release/windows/choco/sqlcmd.nuspec:6]: no, the file can reference the version as a variable, and the pack command that generates the package from the nuspec can set the variable

**Pull request closed without merging**